code/modules/heavy_vehicle/equipment/cult.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/mecha_equipment/mounted_system/soul_javelin
+	name = "mounted soul javelin"
+	desc = "A heavy duty daemon shardlauncher, not for the faint of heart."
+	icon_state = "mecha_souljavelin"
+	holding_type = /obj/item/gun/energy/rifle/cult/mounted
+	equipment_delay = -8
```

Comment:
Is having such a highly negative `equipment_delay` maybe a problem? 

This is a delay of .7 seconds I think? But it could be .2 seconds if they somehow get it on a `/obj/item/mech_component/manipulators/combat` mech.

Do you think we should have a minimum [coded in here](https://github.com/Aurorastation/Aurora.3/blob/885afc07fece63a4c1384cfd740f2d0286a8816a/code/modules/heavy_vehicle/mech_interaction.dm#L142)? 

(Okay so 0.1 is technically already a minimum, but I reckon that is far too low?)